starcli/terminal_size.py

```diff
@@ -1,11 +1,8 @@
 """ starcli.terminal_size """
+from shutil import get_terminal_size
 
 
 def terminal_size():
     """ Fetches the terminal size and returns it """
-    import fcntl, termios, struct
-
-    th, tw, hp, wp = struct.unpack(
-        "HHHH", fcntl.ioctl(0, termios.TIOCGWINSZ, struct.pack("HHHH", 0, 0, 0, 0))
-    )
-    return tw, th
+    term = get_terminal_size()
```

Comment:
Ah, since this function is fairly simple, we should put this line (7) into starcli/layouts.py directly and use `shutil.get_terminal_size` from there

Reply:
Yeah.. it makes sense to add it directly..